game/nintendo_mii/gen1_wii_mii.ksy

```diff
@@ -0,0 +1,250 @@
+meta:
+  id: gen1_wii_mii
+  title: Wii Mii character data
+  file-extension:
+    - mii
+    - mae
+    - miigx
+    - rcd
+    - rsd
+  endian: be
```

Comment:
Another side-note for @fudgepop01:
It appears `meta/bit-endian` is not supported by the VS Code plugin. Adding this will give me a permenant error while I work in VS Code after I add it. That's a bit irritating.